stable/bblfshd-sidecar/values.yaml

```diff
@@ -3,39 +3,13 @@
 # Declare variables to be passed into your templates.
 image:
   repository: bblfsh/bblfshd
-  tag: v2.13.0
+  tag: v2.14.0-drivers
   pullPolicy: IfNotPresent
 
 drivers:
-  install: true
-  languages:
-    python:
-      repository: bblfsh/python-driver
-      tag: latest
-    java:
-      repository: bblfsh/java-driver
-      tag: latest
-    javascript:
-      repository: bblfsh/javascript-driver
-      tag: latest
-    typescript:
-      repository: bblfsh/typescript-driver
-      tag: latest
-    bash:
-      repository: bblfsh/bash-driver
-      tag: latest
-    ruby:
-      repository: bblfsh/ruby-driver
-      tag: latest
-    go:
-      repository: bblfsh/go-driver
-      tag: latest
-    php:
-      repository: bblfsh/php-driver
-      tag: latest
-    csharp:
-      repository: bblfsh/csharp-driver
-      tag: latest
+  install: false
+  languages: {}
+
```

Comment:
Can you add a comment with an example on how to configure this part. I'm sure it will be helpful to avoid inspecting the codew